dracut/anaconda-lib.sh

```diff
@@ -26,6 +26,10 @@ config_get() {
             \[*\]*) cursec="${line#[}"; cursec="${cursec%%]*}" ;;
             *=*) k="${line%%=*}"; v="${line#*=}" ;;
         esac
+        # trim leading and trailing whitespace characters
+        k=$(echo "$k" | sed 's/^ *//;s/ *$//')
```

Comment:
Sorry to be nitpicking, but the commit message and commit is talking about _whitespace_ characters. But it seems sed deals only with _space_ characters here?

Reply:
Good point. Fixed.